code/modules/jobs/job_types/coroner.dm

```diff
@@ -47,14 +47,14 @@
 	backpack_contents = list(
 		/obj/item/storage/box/bodybags = 1,
 		/obj/item/autopsy_scanner = 1,
-		/obj/item/reagent_containers/cup/bottle/formaldehyde = 1,
-		/obj/item/reagent_containers/dropper = 1,
+		/obj/item/storage/medkit/coroner/large = 1,
 	)
 
-	belt = /obj/item/modular_computer/pda/medical
+	belt = /obj/item/modular_computer/pda/coroner
 	ears = /obj/item/radio/headset/headset_srvmed
 	gloves = /obj/item/clothing/gloves/latex/coroner
-	l_pocket = /obj/item/toy/crayon/white
+	l_pocket = /obj/item/clipboard //white crayon is in coroner's kit now, and this is good for storing autopsy reports and taking notes
```

Comment:
comments are meant to express the current state of the game, not reference where stuff was before

Reply:
especially when this job was added like yesterday

Reply:
oh okay, will remove that then!